plugin/fzf.vim

```diff
@@ -844,6 +844,9 @@ endif
 function! s:popup(opts) abort
   " Size and position
   let width = min([max([0, float2nr(&columns * a:opts.width)]), &columns])
+  if &ambiwidth == 'double'
```

Comment:
Let's calculate `ambidouble` before this line and use it (because `x % 1` is always 0)

```vim
let width += width % ambidouble
```

Reply:
thanks